Doc/add a description of Flatten (#447)
open-spaced-repetition/fsrs4anki-helper: README.md

[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)

FSRS Helper is an Anki add-on that supports [FSRS](https://github.com/open-spaced-repetition/fsrs4anki) algorithm. It has six main features:

- **Reschedule** cards based on their entire review histories.
- **Postpone** a selected number of due cards.
- **Advance** a selected number of undue cards.
- **Balance** the load during rescheduling (based on fuzz).
- Less Anki on **Easy Days** (such as weekends) during rescheduling (based on load balance).
- **Disperse** Siblings (cards with the same note) to avoid interference & reminder.
- **Flatten** future due cards to a selected number of reviews per day.

# Requirements

> - For Anki version in 2.1.55 - 2.1.66 (maintenance is stopped)

Reviewer: ```suggestion
- For Anki version in 2.1.55 - 2.1.66 (no longer maintained)
```

  - Enable V3 Scheduler
  - FSRS4Anki version >= 3.0.0

> - For Anki version >= 23.10

Reviewer: Move this above the 2.1.55-2.1.66 part. I believe that the latest version should always be at the top, especially if the older version is no longer maintained.

  - Enable built-in FSRS (follow this [tutorial](https://github.com/open-spaced-repetition/fsrs4anki/blob/main/docs/tutorial.md))

>   - Remove FSRS4Anki custom scheduling code if you used it before

Reviewer: ```suggestion
  - Remove FSRS4Anki custom scheduling code if you have used it before
```

![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/32575846/b33460f4-d7db-4c8f-b9d0-d0193f2d1f54)

## Flatten

> You can the maximum number of reviews per day for future due cards. The Helper will find out the cards that exceed the limit and postpone them to the future.

Reviewer: ```suggestion
You can set the maximum number of reviews per day for future days. The Helper add-on will find out the days that exceed the limit and postpone the extra cards to the future.
```
